include/LogCabin/Client.h

```diff
@@ -113,6 +113,11 @@ struct ConfigurationResult {
          * unavailable.
          */
         BAD = 2,
+        /**
+         * The reconfiguration could not be completed in the requested
+         * timeframe.
```

Comment:
Ah, interesting. I would assume that `\since` applies to the entire surrounding `/** ... */` comment, in this case, scoped to the enum value.